th/2013/spring.json

```diff
@@ -0,0 +1,919 @@
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Akiraz-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Anime@it-fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "DEATH-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Endless-Fs"
+			}, {
+				"name": "Onigiri-Fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "idoli-z"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Kamisama-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "MTM-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Muttsurini-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Muv-Subs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "NAFS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "OTK-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Outlaw"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "StressSyndrome-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Takob"
+			}, {
+				"name": "XilvaliA"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "TSs-Sub"
+			}]
+		}]
+	}, {
+		"name": "Devil Survivor 2 The Animation",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/11/47191.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "Bless"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "DAI-TAKU"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Darkelf"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "DEATH-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Frontier-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "MTM-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Onigiri-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "ShiningMoon-FS"
+			}]
+		}]
+	}, {
+		"name": "Haiyore! Nyaruko-san W",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/10/47533.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "ACE-Z"
+			}, {
+				"name": "MunashiiKaze-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "BDP-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Kimoochi-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "MTM-FS"
+			}]
+		}]
+	}, {
+		"name": "Hayate no Gotoku! Cuties",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/10/48953.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "Anime@it-fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Artstyle Fansub"
+			}, {
+				"name": "Shoutcaster"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Ayumu-FS"
+			}, {
```

Comment:
Can you check if "NTCS-Fs" did two co sub at Hayate (one with Ayumu-FS and one with DJ-FS)?